lib/crowbar/client/request/node/action.rb

```diff
@@ -18,6 +18,9 @@ module Crowbar
   module Client
     module Request
       module Node
+        #
+        # Implementation for the node action request
```

Comment:
No, I won't do that. This will be obvious within the yard documentation as you can see all child classes over there.